content/pages/homepage.html

```diff
@@ -38,7 +38,7 @@ <h1 class="ui huge header">
                         <i class="fad fa-play icon"></i>
                         Get started
                       </a>
-                      <a href="/product.html" class="ui large stackable button">Learn more</a>
+                      <a href="//" class="ui large stackable button">Learn more</a>
```

Comment:
Guessing this should be `/about/`

```suggestion
                      <a href="/about/" class="ui large stackable button">Learn more</a>
```